_source/logzio_collections/_metrics-sources/kubernetes.md

```diff
@@ -105,13 +105,19 @@ kubectl --namespace=kube-system create secret generic logzio-metrics-secret \
 
 ##### Store your cluster details
 
-Paste the kube-state-metrics namespace and port in your text editor.
-You can find them by running this command.
+Paste the kube-state-metrics namespace in your text editor.
+You can find it by running this command.
 
 ```shell
 kubectl get service --all-namespaces | grep -E 'kube-state-metrics|NAMESPACE'
 ```
 
+Next, run a command to find the kube-state-metrics port under `http-metrics`.
+
+```shell
+kubectl get service -n <KUBE-STATE-METRICS-NAMESPACE> kube-state-metrics -o yaml
+```
+
```

Comment:
```suggestion
Look up the following cluster details and copy them into your text editor or another convenient place for quick reference. 

Run the following command to get your <kube-state-metrics-namespace>.

```shell
kubectl get service --all-namespaces | grep -E 'kube-state-metrics|NAMESPACE'
```

Next, run a command to get your kube-state-metrics port. After you run the command, it will appear in the output under `http-metrics`. 

```shell
kubectl get service -n <KUBE-STATE-METRICS-NAMESPACE> kube-state-metrics -o yaml
```

```